bundles/org.openhab.binding.helios/src/main/java/org/openhab/binding/helios/internal/handler/HeliosHandler221.java

```diff
@@ -979,6 +981,11 @@ private List<RESTPort> getPorts() {
                             logger.trace("No events were retrieved");
                         }
                     }
+                } catch (ProcessingException e) {
+                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
+                    logger.warn("An underlying exception forced the Helios IP Vario to go offline : '{}'",
```

Comment:
This logging is redundant, since the status update (including the exception message) will already be logged. If you need the full stack trace, you could reduce to **debug** or **trace**.